templates/elastic-beanstalk.cfn.yml

```diff
@@ -3,17 +3,19 @@ Description:  AWS Elastic Beanstalk app - built by AWS Startup Kit
 
 # App stack creation prerequisites:  first create a VPC stack, then a DB stack.
 
+Description: Elastic Beanstalk
```

Comment:
These descriptions should be fleshed out using information either in the README or that you add to the README if you follow my paths advice.

Reply:
They look really good the way they are in the console. The wrapper stacks look like: "VPC + Bastion + Elastic Beanstalk + Database" The nested stacks that are launched then say, "Elastic Beanstalk"